pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = -rsxX --reuse-db --tb=native
```

Comment:
I copied these from fjord. I don't know what `X` does and I don't think `x` is actually causing it to exit fast. So it's possible this has no effect on things.

I'll look that unless someone knows and can suggest better things here.